CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 4.12.2 (2022-01-20)
```

Comment:
Shouldn't this PR update the `package.json` file's version property to 4.12.2?

Reply:
Doh!